doc/sphinx/proof-engine/tactics.rst

```diff
@@ -676,7 +676,19 @@ Applying theorems
 
    Setting :opt:`Debug` ``"unification"`` enables printing traces of
    unification steps used during elaboration/typechecking and the
-   :tacn:`refine` tactic. ``"ho-unification"`` prints information
+   :tacn:`refine` tactic. Each ``Debug: [unification]`` output block
+   is a snapshot of the current problem the unification problem is
+   trying to solve. Given two terms :term:`term1`, :term:`term2`, the
```

Comment:
I don't think `:term:` is what you want here, it's for linking to glossary entries https://rocq-prover.org/doc/master/refman/std-glossindex.html